frontend/language/src/nb.json

```
@@ -16,7 +16,9 @@
   "administration.activity": "Aktivitet",
   "administration.administration": "Om appen",
   "administration.app_environments_error": "Kunne ikke laste inn statusen for denne applikasjonen. Prøv igjen senere.",
+  "administration.app_logs_created": "{createdBy} dato: {createdDateTime}",
```

Dette blir vel slik hvis det skal stemme helt med designet:
```suggestion
  "administration.app_logs_created": "({createdBy}) Dato: {createdDateTime}",
```
Når jeg ser på dette en gang til nå, gir det egentlig ikke helt mening at dette er samme tekstblokk. Bruker og dato burde enten være mer adskilt i designet (med andre effekter enn parenteser), slik at det naturlig blir to tekstblokker, eller så burde de fremstå slik at det hele gir mening som en sammenhengende tekst (typ `Opprettet {createdDateTime} av {createdBy}`). Jeg heller mot det siste, men dette blir vel fort en diskusjon vi må ta med designavdelingen.